api/src/org/apache/cloudstack/api/command/admin/host/ListHostsCmd.java

```diff
@@ -206,7 +206,7 @@ public void execute() {
         } else {
             Pair<List<? extends Host>, Integer> result;
             Ternary<Pair<List<? extends Host>, Integer>, List<? extends Host>, Map<Host, Boolean>> hostsForMigration =
-                _mgr.listHostsForMigrationOfVM(getVirtualMachineId(), this.getStartIndex(), this.getPageSizeVal());
+                _mgr.listHostsForMigrationOfVM(getVirtualMachineId(), this.getStartIndex(), this.getPageSizeVal(), null);
```

Comment:
@koushik-das One of the changes in this commit was to add keyword as part of the listHostsForMigrationOfVM API. Earlier in this place the keyword was not getting passed down. Since it wasnt passed earlier, to keep this call the same, passing null as input. Hope this explains.